eeps/eep-0048.md

```diff
@@ -101,32 +101,35 @@ the type specification below:
 
     {docs_v1,
      Anno :: erl_anno:anno(),
-     Language :: atom(),
+     BeamLanguage :: atom(),
      Format :: mime_type(),
-     ModuleDoc :: binary() | none | hidden,
+     ModuleDoc :: %{DocLanguage := DocString} | none | hidden,
```

Comment:
Isn't `%{...}` the Elixir notation of maps, in which case it should be replaced with `#{...}`?

Reply:
Haha, yes... too much context switching.